packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -83,6 +83,12 @@ class CupertinoColors {
   /// This is SystemGrayColor in the iOS palette.
   static const Color inactiveGray = Color(0xFF8E8E93);
 
+  /// Used in iOS 13 for the "off" label when accessibility switch labels are
+  /// enabled.
+  ///
+  /// Value derived from a screenshot of a switch with on/off labeling.
+  static const Color contrastingGray = Color(0xFFB8B8B8);
```

Comment:
Is this a widely used color that is worth exposing? Also the color value I got from inspecting the native component using Xcode is `0xFFB3B3B3` (White:0.7 Alpha:1).

Reply:
I don't think so -- I just felt guilty adding a color constant to switch.dart 😜 

If that seems reasonable, I'll remove the constant from here and add it to the constants section (as 0xFFB3B3B3 -- I used the OS X color meter utility, but I think you're 100% right that Xcode is more accurate).